dgbpy/keras_classes.py

```diff
@@ -252,13 +253,29 @@ class variable.
           if issubclass(c, UserModel) & (c is not UserModel):
             mlm.append(c())
 
-    for _, name, ispkg in pkgutil.iter_modules():
-      if name.startswith('mlmodel_keras'):
-        module = importlib.import_module(name)
-        clsmembers = inspect.getmembers(module, inspect.isclass)
-        for (_, c) in clsmembers:
-          if issubclass(c, UserModel) & (c is not UserModel):
-            mlm.append(c())
+    home_path = os.path.expanduser('~')
+    py_settings_path = os.path.join(home_path, '.od','settings_python')
```

Comment:
have a look at odpy.get_base_datadir()
You may be able to use the following:
settsfnm = odpy.get_settings_filename( 'settings_python' )
read_from_iopar( settsfnm, pattern )

Reply:
Okay, great thanks.
Will give it a check